app/src/main/java/com/greenmonkeys/handmade/ConfirmPayoutActivity.kt

```diff
@@ -0,0 +1,44 @@
+package com.greenmonkeys.handmade
+
+import android.app.Activity
+import android.content.Intent
+import android.graphics.Bitmap
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import android.widget.TextView
+
+const val CAPTURE_SIGNATURE_INTENT = 1
+
+class ConfirmPayoutActivity : AppCompatActivity() {
+    lateinit var amount: TextView
+    lateinit var backButton: Button
+    lateinit var signButton: Button
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_confirm_payout)
+
+        val payoutAmount = intent.getDoubleExtra("amount", 0.0)
+        amount = findViewById(R.id.activity_confirm_payout_amount)
+        amount.text = "$${payoutAmount}"
```

Comment:
You could concatenate these strings to make this one more readable and less prone to errors if edited.